XLS-47d-PriceOracles/README.md

```diff
@@ -62,17 +62,18 @@ The `PriceOracle` ledger entry represents the `PriceOracle` object on XRPL ledge
 
 ##### The `PriceOracle` Object ID Format
 
-We compute the `PriceOracle` object ID, a.k.a., `OracleID`, as the SHA-512Half of the following values, concatenated in order:
+We compute the `PriceOracle` object ID as the SHA-512Half of the following values, concatenated in order:
 
 - The Oracle space key (0x52)
-- The Owner Account ID
-- The Oracle Sequence. This field must be passed to the transactions and it describes a unique Price Oracle sequence for the given account.
+- The Owner Account ID, `Account`.
```

Comment:
`Account` here but `Owner` in the next paragraph. I was guessing it should be `Owner` in both places (as it is in the fields table above and example below), but it is `Account` in the transactions. Can we unify? Should we? I'm guessing we use `Account` in the transactions because all transactions must have that field? Is that true?

Reply:
Correct, `Account` is used in the transactions. I'll update to `Owner` outside of the transactions.